app/build.gradle

```diff
@@ -181,10 +195,10 @@ dependencies {
 
     debugImplementation "androidx.fragment:fragment-testing:$fragment_version"
 
-    implementation "androidx.camera:camera-core:1.1.0-alpha02"
-    implementation "androidx.camera:camera-camera2:1.1.0-alpha02"
-    implementation "androidx.camera:camera-lifecycle:1.1.0-alpha02"
-    implementation "androidx.camera:camera-view:1.0.0-alpha22"
```

Comment:
I think last time I tried to do that bump it broke something, going to check real quick

Reply:
nevermind